presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -283,14 +284,14 @@ public static ConnectorPageSource createParquetPageSource(
                     blocks.add(block);
                     blockStarts.add(nextStart);
                     blockIndexStores.add(columnIndexStore.orElse(null));
-                    hiveFileContext.incrementCounter("parquet.blocksRead", 1);
-                    hiveFileContext.incrementCounter("parquet.rowsRead", block.getRowCount());
-                    hiveFileContext.incrementCounter("parquet.totalBytesRead", block.getTotalByteSize());
+                    hiveFileContext.incrementCounter("parquet.blocksRead", RuntimeUnit.NONE, 1);
```

Comment:
Please do a static import and use NONE directly.
`hiveFileContext.incrementCounter("parquet.blocksRead", NONE, 1);`

Reply:
done

Reply:
done